packages/dialect-react-ui/components/Notifications/screens/NewSettings/Telegram.tsx

```diff
@@ -131,7 +129,7 @@ const Telegram = () => {
     if (!dappAddress) {
       return defaultBotUrl;
     }
-    const dapp = dapps[dappAddress.toBase58()];
+    const dapp = dapps[dappAddress.toString()];
```

Comment:
Since `dappAddress` is of `AccountAddress` type, I'm assuming it's essentially a `string` already. So I think we don't need to call the `toString()` method here. Please correct me if I'm wrong.